Failed test_get_package_tuple_compiled: AttributeError: module 'zlib' has no attribute '__file__'

**Describe the bug**
When building `rope` package, test `test_get_package_tuple_compiled` fails on my system due to unrelated error (assertion on `zlib` module).

**To Reproduce**

After cloning the GitHub project, execute the following commands in the source dir:

```
python3 setup.py config
python3 setup.py build
python3 -m pytest
```

The following error will occur:
```
==================================== ERRORS ====================================
______________ ERROR at setup of test_get_package_tuple_compiled _______________

    @pytest.fixture
    def zlib_path():
        import zlib
    
>       yield pathlib.Path(zlib.__file__)
E       AttributeError: module 'zlib' has no attribute '__file__'

ropetest/contrib/autoimport/conftest.py:58: AttributeError
```

I am not sure why the test assumes that `zlib.__file__` should exist:

```
% python3
Python 3.10.5 (main, Jun  8 2022, 09:26:22) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import zlib
>>> zlib.__file__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'zlib' has no attribute '__file__'. Did you mean: '__name__'?
>>> zlib
<module 'zlib' (built-in)>
>>>
```

**Editor information (please complete the following information):**
 - OS version: Debian Unstable/Sid
 - Python version: 3.10.5
 - Rope version: 1.2.0

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,7 @@
 ## Bug fixes
 
 - #496,#497 Add MatMul operator to patchedast
+- #495 Fix autoimport collection for compiled modules
 
 # Release 1.2.0
 

--- a/rope/contrib/autoimport/utils.py
+++ b/rope/contrib/autoimport/utils.py
@@ -28,6 +28,9 @@ def get_package_tuple(
         if package_name.endswith(".so"):
             package_name = package_name.split(".")[0]
             package_type = PackageType.COMPILED
+        elif package_name.endswith(".pyd"):
+            package_name = package_name.split(".")[0]
+            package_type = PackageType.COMPILED
         elif package_name.endswith(".py"):
             package_name = package_path.stem
             package_type = PackageType.SINGLE_FILE

--- a/ropetest/contrib/autoimport/conftest.py
+++ b/ropetest/contrib/autoimport/conftest.py
@@ -52,7 +52,7 @@ def build_path():
 
 
 @pytest.fixture
-def zlib_path():
-    import zlib
+def compiled_lib():
+    import _sqlite3
 
-    yield pathlib.Path(zlib.__file__)
+    yield "_sqlite3", pathlib.Path(_sqlite3.__file__)

--- a/ropetest/contrib/autoimport/utilstest.py
+++ b/ropetest/contrib/autoimport/utilstest.py
@@ -56,8 +56,8 @@ def test_get_package_tuple_typing(typing_path):
     ) == utils.get_package_tuple(typing_path)
 
 
-@pytest.mark.skipif(platform == "win32", reason="Windows doesn't have compiled modules")
-def test_get_package_tuple_compiled(zlib_path):
+def test_get_package_tuple_compiled(compiled_lib):
+    lib_name, lib_path = compiled_lib
     assert Package(
-        "zlib", Source.STANDARD, zlib_path, PackageType.COMPILED
-    ) == utils.get_package_tuple(zlib_path)
+	lib_name, Source.STANDARD, lib_path, PackageType.COMPILED
+    ) == utils.get_package_tuple(lib_path)

